.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 /addons/vscode/out/
 node_modules/
+.vscode/
```

Comment:
We do want to track this directory, since `tasks.json` and `launch.json` are used to compile and prepare the extension during development.